cmd/distrogen/registry.go

```diff
@@ -140,13 +185,15 @@ type otelComponentVersion struct {
 // RegistryComponent is the type used as a basis for Registry.
 // It contains all the information needed to output a
 type RegistryComponent struct {
-	GoMod         *GoModuleID `yaml:"gomod"`
-	Import        string      `yaml:"import,omitempty"`
-	Name          string      `yaml:"string,omitempty"`
-	Path          string      `yaml:"path,omitempty"`
-	Stable        bool        `yaml:"stable,omitempty"`
-	StartRevision string      `yaml:"start_revision,omitempty"`
-	DocsURL       string      `yaml:"docs_url"`
+	Name string `yaml:"-"`
+
+	GoMod         *GoModuleID               `yaml:"gomod"`
+	Import        string                    `yaml:"import,omitempty"`
+	Path          string                    `yaml:"path,omitempty"`
+	Stable        bool                      `yaml:"stable,omitempty"`
+	StartRevision string                    `yaml:"start_revision,omitempty"`
+	DocsURL       string                    `yaml:"docs_url,omitempty"`
+	Releases      RegistryComponentReleases `yaml:"releases,omitempty"`
```

Comment:
idk if I added this or you did, either way we can remove it for now since I'm handling it in my PR.